docs/topics/benchmarking.rst

```diff
@@ -12,11 +12,15 @@ to get an idea of how Scrapy performs in your hardware, in order to have a
 common baseline for comparisons. It uses a simple spider that does nothing and
 just follows links.
 
-To run it use::
+To run it use:
+
+.. code-block:: bash
 
     scrapy bench
 
-You should see an output like this::
+You should see an output like this:
+
+::.. code-block:: bash
```

Comment:
The documentation is [currently failing](https://travis-ci.org/scrapy/scrapy/jobs/609184273?utm_medium=notification&utm_source=github_status) due to this.